components/browser/menu/src/test/java/mozilla/components/browser/menu/item/BrowserMenuImageTextTest.kt

```diff
@@ -0,0 +1,93 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package mozilla.components.browser.menu.item
+
+import android.content.Context
+import android.support.v7.widget.AppCompatImageView
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.TextView
+import androidx.test.core.app.ApplicationProvider
+import mozilla.components.browser.menu.BrowserMenu
+import mozilla.components.browser.menu.R
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertNull
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.mock
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class BrowserMenuImageTextTest {
+
+    private val context: Context = ApplicationProvider.getApplicationContext()
```

Comment:
Is it guaranteed that robolectric will use the same application context for all test methods? Otherwise I'd turn this into:

```Kotlin
    private val context: Context
        get() = ApplicationProvider.getApplicationContext()
```